src/main/java/net/dzikoysk/funnyguilds/listener/BlockFlow.java

```diff
@@ -0,0 +1,27 @@
+package net.dzikoysk.funnyguilds.listener;
+
+import net.dzikoysk.funnyguilds.basic.guild.Region;
+import net.dzikoysk.funnyguilds.basic.guild.RegionUtils;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BlockFromToEvent;
+
+public class BlockFlow implements Listener {
+
+    @EventHandler
+    public void blockFlow(BlockFromToEvent event) {
+
+        Block block = event.getBlock();
+        if (block.getType() == Material.LAVA || block.getType() == Material.WATER) {
```

Comment:
Maybe use https://papermc.io/javadocs/paper/1.16/org/bukkit/block/Block.html#isLiquid-- instead? I'm not entirely sure if this method is supported in legacy versions, e.g. 1.8.8

Reply:
Good idea